scientific_library/contrib/tvb/contrib/scripts/datatypes/head.py

```diff
@@ -2,58 +2,65 @@
 
 import os
 
+from tvb.basic.logger.builder import get_logger
+from tvb.basic.neotraits.api import HasTraits, Attr
 from tvb.contrib.scripts.datatypes.base import BaseModel
 from tvb.contrib.scripts.datatypes.connectivity import Connectivity
 from tvb.contrib.scripts.datatypes.local_connectivity import LocalConnectivity
-from tvb.contrib.scripts.datatypes.projections import ProjectionSurfaceEEG, ProjectionSurfaceSEEG, \
-    ProjectionSurfaceMEG
+from tvb.contrib.scripts.datatypes.projections import ProjectionSurfaceEEG, ProjectionSurfaceSEEG, ProjectionSurfaceMEG
 from tvb.contrib.scripts.datatypes.region_mapping import CorticalRegionMapping, SubcorticalRegionMapping, \
     RegionVolumeMapping
 from tvb.contrib.scripts.datatypes.sensors import SensorsEEG, SensorsSEEG, SensorsMEG
 from tvb.contrib.scripts.datatypes.structural import T1, T2, Flair, B0
 from tvb.contrib.scripts.datatypes.surface import Surface, CorticalSurface, SubcorticalSurface
 from tvb.contrib.scripts.utils.file_utils import insensitive_glob
-from tvb.basic.logger.builder import get_logger
-from tvb.basic.neotraits.api import HasTraits, Attr
 from tvb.datatypes.connectivity import Connectivity as TVBConnectivity
 from tvb.datatypes.cortex import Cortex
 from tvb.datatypes.local_connectivity import LocalConnectivity as TVBLocalConnectivity
 from tvb.datatypes.projections import EEG_POLYMORPHIC_IDENTITY, SEEG_POLYMORPHIC_IDENTITY, MEG_POLYMORPHIC_IDENTITY
 from tvb.datatypes.projections import ProjectionMatrix as TVBProjectionMatrix
+from tvb.datatypes.projections import ProjectionSurfaceEEG as TVBProjectionSurfaceEEG
+from tvb.datatypes.projections import ProjectionSurfaceMEG as TVBProjectionSurfaceMEG
+from tvb.datatypes.projections import ProjectionSurfaceSEEG as TVBProjectionSurfaceSEEG
 from tvb.datatypes.region_mapping import RegionMapping as TVBRegionMapping
 from tvb.datatypes.region_mapping import RegionVolumeMapping as TVBRegionVolumeMapping
 from tvb.datatypes.sensors import Sensors as TVBSensors
+from tvb.datatypes.sensors import SensorsEEG as TVBSensorsEEG
+from tvb.datatypes.sensors import SensorsInternal as TVBSensorsInternal
+from tvb.datatypes.sensors import SensorsMEG as TVBSensorsMEG
 from tvb.datatypes.structural import StructuralMRI as TVBStructuralMRI
+from tvb.datatypes.surfaces import CorticalSurface as TVBCorticalSurface
 from tvb.datatypes.surfaces import Surface as TVBSurface
+from tvb.simulator.plot.config import CONFIGURED
 from tvb.simulator.plot.utils import raise_value_error
 
+logger = get_logger(__name__)
 
-class Head(HasTraits):
-    logger = get_logger(__name__)
 
+class Head(HasTraits):
```

Comment:
`HasTraits` already has a `self.log` attribute, which we could use.

Reply:
Therefore there is no need for the logger property there.